src/SignaturePad.InkPresenter.Shared/InkPresenter.cs

```diff
@@ -84,7 +96,11 @@ private NativeRect DirtyRect
 				var w = Math.Abs (dirtyRectRight - dirtyRectLeft);
 				var h = Math.Abs (dirtyRectBottom - dirtyRectTop);
 				var half = StrokeWidth / 2f;
+#if NET471
+				return new NativeRect ((int)(x - half), (int)(y - half), (int)(w + StrokeWidth), (int)(h + StrokeWidth));
+#else
```

Comment:
If we are using `RectangleF`, then this won't be needed.

Reply:
+